dev/com.ibm.ws.session/src/com/ibm/ws/session/SessionManagerConfig.java

```diff
@@ -1166,31 +1174,44 @@ public final void setSessionCookieSameSite(SameSiteCookie sameSite) {
     
 
     //Servlet 6.0 - updated to use interface
+    // if web.xml cookie-config found, method will update the this SMC.cookieConfig with the webApp cookieConfig
     public void updateCookieInfo(SessionCookieConfig scc) {
         if (scc != null) {
             SessionCookieConfigImpl sccImpl = (SessionCookieConfigImpl) scc;
             
-            if (scc.getComment() != null) {
-                this.setSessionCookieComment(scc.getComment());
+            if (TraceComponent.isAnyTracingEnabled() && LoggingUtil.SESSION_LOGGER_CORE.isLoggable(Level.FINER)) {
+                LoggingUtil.SESSION_LOGGER_CORE.log(Level.FINE, methodClassName + " updateCookieInfo , replace [" + cookieConfig +"] with ["+ scc + "] for this -> " + this);
+            }
+
+            //Servlet 6 update - webAppConfig SCC should populate this SMC. Any null/unset webAppConfig's SCC field is
+            //replaced with the current SMC's field before webAppConfig SCC replacing the SMC's SCC
+            //The webAppConfig SCC can be an instance of SCC 6 and may have addition attributes (already parsed from web.xml or set later by application via SCI)
+            //Also, a little later after this method is called, the SessionContextRegistryImpl will set both Session SMC and webAppConfig
+            //to the same SCC anyway (around line 318)
```

Comment:
Should you add a method name here etc as line number tend to change pretty frequently?